pageserver/src/tenant/timeline/init.rs

```diff
@@ -80,51 +102,41 @@ pub(super) enum Decision {
     /// The layer is present locally, but local metadata does not match remote; we must
     /// delete it and treat it as evicted.
     UseRemote {
-        local: LayerFileMetadata,
+        local: LocalLayerFileMetadata,
         remote: LayerFileMetadata,
     },
     /// The layer is present locally, and metadata matches.
-    UseLocal(LayerFileMetadata),
+    UseLocal(LocalLayerFileMetadata),
 }
 
 /// A layer needs to be left out of the layer map.
 #[derive(Debug)]
 pub(super) enum DismissedLayer {
     /// The related layer is is in future compared to disk_consistent_lsn, it must not be loaded.
     Future {
-        /// The local metadata. `None` if the layer is only known through [`IndexPart`].
-        local: Option<LayerFileMetadata>,
+        /// The path to local layer file. `None` if the layer is only known through [`IndexPart`].
+        local: Option<LocalLayerFileMetadata>,
```

Comment:
Still mentions path. Maybe we can remove the first sentence alltogether?

Reply:
Cleaned up in a1d43d8df